apps/src/lib/levelbuilder/script-editor/scriptEditorRedux.js

```diff
@@ -364,11 +364,12 @@ const serializeLesson = (lesson, levelKeyList) => {
         s.push('variants');
         level.ids.forEach(id => {
           const active = id === level.activeId;
-          s.push(`  ${serializeLevel(levelKeyList, id, level, active)}`);
+          const lines = serializeLevel(levelKeyList, id, level, active);
+          s = s.concat(lines.map(line => `  ${line}`));
         });
         s.push('endvariants');
```

Comment:
should this line and line 364 not also be `concat`?

Reply:
I think either would work because s.push and s = s.concat do the same thing when passed a string. serializeLevel is different because it (confusingly) returns an array.